openshift/pod-template.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: adviser-pod
+  thoth: 0.1.0
+metadata:
+  name: adviser
+  annotations:
+    description: This is Thoth Core - Adviser
+    openshift.io/display-name: 'Thoth: Adviser'
+    version: 0.3.0
+    tags: poc,thoth,ai-stacks,graph-sync-job
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to run recommendation logic of Thoth to OpenShift.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+
+objects:
+  - apiVersion: v1
+    kind: Pod
+    metadata:
+      generateName: adviser-
+      labels:
+        app: thoth
+        thoth-pod: adviser
+    spec:
+      restartPolicy: Never
+      automountServiceAccountToken: false
+      containers:
+        - name: adviser
+          image: adviser
+          livenessProbe:
+             # Give analyzer 10 minutes to compute results, kill it if it was not able result anything.
+             tcpSocket:
+               port: 80
+             initialDelaySeconds: 600
+             failureThreshold: 1
+             periodSeconds: 10
+          env:
+            - name: THOTH_LOG_ADVISER
+              value: 'INFO'
+            # These are intentionally empty - they will be provided on pod run request.
+            - name: THOTH_ADVISER_OUTPUT
```

Comment:
all ENV are empty strings?!

Reply:
I wanted to have them present explicitly - they will be supplied by user-api based on user input (see comment).